README.md

```diff
@@ -1,4 +1,38 @@
-# Python boilerplate for GitHub tasks
+# Distance class
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+
+Every day you have morning running. You want to store your result. 
+To easy calculating, comparison and printing you result write class
+`Distance`.
+
+`Distance`'s constructor takes only
+one argument `km` and saves it to `self.km`.
+
+For `Distance` class you should implement such magic
+methods:
+```python
+__init__
+distance = Distance(20) # distance.km == 20
+__str__
+print(distance) # "Distance: 20 kilometers."
```

Comment:
Could you add here also __repr__ method:
repr(distance)  # "Distance(km=20)"

Reply:
Also before inline comments like here:
```python
print(distance) # "Distance: 20 kilometers."
```
Try to add 2 spaces here:
![image](https://user-images.githubusercontent.com/48014331/149350435-86360276-7a97-4f7c-8aff-a185dfa08305.png)